source/jsonnet/england-wales/individual/blocks/identity-and-health/national_identity.jsonnet

```diff
@@ -2,30 +2,15 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import 'rules.libsonnet';
 
 local nonProxyTitle = 'How would you describe your national identity?';
+local nonProxyDescription = 'This relates to where you feel you belong, such as the country or countries you think of as home. <br><br> This could be different from your citizenship or ethnic group.';
```

Comment:
Can you remove spaces from before and after **br** tags? They look like this in html at the moment:
<img width="585" alt="Screenshot 2020-03-23 at 13 26 24" src="https://user-images.githubusercontent.com/53475968/77321351-fcb3da80-6d09-11ea-9caf-ce1259936c8d.png">
```suggestion
local nonProxyDescription = 'This relates to where you feel you belong, such as the country or countries you think of as home.<br><br>This could be different from your citizenship or ethnic group.';
```